packages/paste-website/src/components/tokens-list/category-section/FontSizeCallout.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+import {CalloutText, Callout, CalloutTitle} from '../../callout';
+
+// @TODO add when we add category meta etc.
+export const FontSizeCallout: React.FC = () => (
+  <Callout>
+    <CalloutTitle as="h4">Heads up about font sizes in Paste!</CalloutTitle>
+    <CalloutText>
+      Font-sizes in our Paste tokens use rem values. Unlike em values which are relative to their parent element, rem
+      values are relative to the html element. If you aren&apos;t using the Theme.Provider component, you must set
```

Comment:
suggestion (non-blocking): It might be helpful to link the theme package part of our docs site (core/theme-package#theme-provider) here.